### ISSUE TITLE
Error: 'System.QueryException: Implementation restriction...'

### ISSUE THREAD

**Issue by danielsokolowski** [afawcett/apex-sobjectdataloader#9]
Wondering if you can point me in the right direction where to start troubleshooting the below error - the lugs are cut off:

```
CollaborationGroupRecord requires a filter by a single Id, CollaborationGroupId or Record using the equals operator
```

![sobject-data-loader-serialize-error](https://cloud.githubusercontent.com/assets/273413/6606290/e2c37826-c80c-11e4-985a-046d13011f86.png)

**Comment by afawcett** [comment]
Yes this has been raised before, see a workaround in this issue [here](https://github.com/afawcett/apex-sobjectdataloader/issues/8).
